recipes/acado/all/conanfile.py

```diff
@@ -39,9 +39,11 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
-        if self.settings.compiler == "Visual Studio" and \
-           "MT" in self.settings.compiler.runtime and self.options.shared:
-            raise ConanInvalidConfiguration("Acado does not support MT runtime + shared library.")
+        if self.settings.compiler == "Visual Studio" and self.options.shared:
+            if "MT" in self.settings.compiler.runtime:
+                raise ConanInvalidConfiguration("Acado does not support MT runtime + shared library.")
+            if "MD" in self.settings.compiler.runtime:
+                raise ConanInvalidConfiguration("Acado does not support MD runtime + shared library.")
```

Comment:
This blocks windows completely

Reply:
Have to check if acado supports shared library on Windows.

Reply:
ok, do you then know any other way to make it work? I have no Windows setup to debug this:
```
[HOOK - conan-center.py] post_package(): ERROR: [SHARED ARTIFACTS (KB-H015)] Package with 'shared' option did not contains any shared artifact (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H015)
```